lib/src/iterable_extensions.dart

```diff
@@ -0,0 +1,519 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Extension methods on iterables.
+
+import 'package:collection/src/utils.dart';
+
+import "algorithms.dart";
+
+/// Extensions that apply to all iterables.
```

Comment:
If the only thing we can say is to repeat the signature, we should omit the comment. Same thing for all the docs on the extensions in this file.